src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.ts

```diff
@@ -14,11 +16,48 @@ export class InstructorHelpCoursesSectionComponent extends InstructorHelpSection
 
   readonly supportEmail: string = environment.supportEmail;
 
-  constructor() {
+  isAddStudentsCollapsed: boolean = false;
+  isSizeLimitCollapsed: boolean = false;
+  isNoTeamsCollapsed: boolean = false;
+  isSectionsCollapsed: boolean = false;
+  isEnrollSectionsCollapsed: boolean = false;
+  isAddInstructorCollapsed: boolean = false;
+  isEditInstructorCollapsed: boolean = false;
+  isInstructorAccessCollapsed: boolean = false;
+  isPrivilegesCollapsed: boolean = false;
+  isViewStudentsCollapsed: boolean = false;
+  isChangeSectionCollapsed: boolean = false;
+  isDisappearedCourseCollapsed: boolean = false;
+  isDelStudentsCollapsed: boolean = false;
+  isArchiveCourseCollapsed: boolean = false;
+  isViewArchivedCollapsed: boolean = false;
+  isCourseUnarchiveCollapsed: boolean = false;
+  isViewDelCollapsed: boolean = false;
+  isRestoreCollapsed: boolean = false;
+  isDelCollapsed: boolean = false;
+  isRestoreAllCollapsed: boolean = false;
+  @Output() collapseStudentEditDetails: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+  constructor(private pageScrollService: PageScrollService,
+              @Inject(DOCUMENT) private document: any) {
     super();
   }
 
   ngOnInit(): void {
+  }
 
+  /**
+   * Scrolls to an HTML element with a given target id.
+   */
+  jumpTo(target: string): boolean {
```

Comment:
For better code quality, I think you can create a parent to do code sharing for `jumpTo`